src/Exception/AdapterConfiguration/AdapterNotConfiguredException.php

```diff
@@ -32,7 +30,6 @@ public function getSolution(): ?string
             References:
             - $queueClass::\$adapter
             - $queueClass::withAdapter()
-            - $factoryClass::get()
```

Comment:
QueueFactory in solution text should be replaced with QueueProviderInterface. I think it would be got to paste here a link to providers configuration docs.

Reply:
On adapted based `AdapterFactoryQueueProvider` only. May be only add him instead `QueueProviderInterface`?

Reply:
No, it's a general exception thrown by the Queue class

Reply:
Added `QueueProviderInterface::class`